lib/endpoints/pivtokens.js

```diff
@@ -473,13 +473,28 @@ function createRecoveryToken(req, res, next) {
 
     const latestToken = req.pivtoken.recovery_tokens[
             req.pivtoken.recovery_tokens.length - 1];
+    const params = {
+        pivtoken: req.pivtoken.guid,
+        recovery_configuration: req.params.recovery_configuration ||
+            latestToken.recovery_configuration
+    };
+
+    if (req.params.token) {
+        params.token = new Buffer(req.params.token, 'base64').toString();
```

Comment:
Would something like `Buffer.from(req.params.token, 'base64').toString()` be preferred? IIUC this is the 'newer' syntax.. and `new Buffer()` is deprecated in newer node versions..

Reply:
Good call, let's modify before merging.